entries/b/britishairways.com.json

```diff
@@ -1,10 +1,12 @@
 {
   "British Airways": {
     "domain": "britishairways.com",
-    "contact": {
-      "facebook": "britishairways",
-      "twitter": "British_Airways"
-    },
+    "tfa": [
+      "sms",
+      "totp",
+      "u2f"
+    ],
+    "notes": "Set-up requires linking one of the following services with your BA account: Qatar, Iberia, Uber",
```

Comment:
I personally didn't encounter this and don't have any of these linked. If it happens for some then I'd suggest changing the wording from "requires" to "may require".